src/utils.js

```diff
@@ -155,7 +157,7 @@ const snakeCaseDict = (data) => {
   const transformedData = {};
   [...Object.entries(data)]
     .forEach(entry => {
-      transformedData.append(snakeCase(entry[0]), entry[1]);
```

Comment:
is it worth a test (even if in a new ticket) since there was a bug that was found here? a nice unit test could be swell! but please don't let this slow this PR down, so better a new ticket maybe (unless it's trivial)

Reply:
a test has been added